ubiconfig/config_types/modules.py

```diff
@@ -9,7 +9,7 @@ def __init__(self, name, stream, profiles=None):
         self.profiles = profiles
 
     def __repr__(self):
-        return self.name
+        return "Module(name=%s)" % self.name
```

Comment:
This is pretty close to eval(repr(x)) == x, but it doesn't quite work since the name part doesn't have quotes.  Same comment for Package.

You would rather need to do e.g.


    "Module(name=%s)" % repr(self.name)

...so it produces `Module(name="foo")` rather than `Module(name=foo)`.

Reply:
+1